alpha/alpha-core/src/main/java/org/apache/servicecomb/saga/alpha/core/Command.java

```diff
@@ -55,14 +55,14 @@
   }
 
   private Command(long id,
-      String serviceName,
-      String instanceId,
-      String globalTxId,
-      String localTxId,
-      String parentTxId,
-      String compensationMethod,
-      byte[] payloads,
-      String status) {
+                  String serviceName,
```

Comment:
Please don't reformat the code that you don't change. 
You can setup your IDE to only reformat the code that VCS changed.
http://servicecomb.apache.org/developers/setup-develop-environment/